index.py

```diff
@@ -81,18 +80,19 @@
     print(f"\nSeems like you entered an {Fore.RED}invalid token{Fore.RESET}. Please enter a valid token (see Github repo for help).")
 
     # Resets the config so that it doesn't use the previous token again
-    config = {}
+    config.clear()
 
 
-# This is used to save the token for the next time you run the bot
-with open("config.json", "w") as f:
-    # Check if 'token' key exists in the config.json file
```

Comment:
It was supposed to write a json file regardless, because if the file is there and they put a different token because the first one failed, it would overwrite and fix the config file. Open() write mode makes a file or overwrites the file, which is perfect for this case.

Reply:
> It was supposed to write a json file regardless, because if the file is there and they put a different token because the first one failed, it would overwrite and fix the config file. Open() write mode makes a file or overwrites the file, which is perfect for this case.

Whoops, totally forgot that could happen. Sorry.